system-monitor@paradoxxx.zero.gmail.com/extension.js

```diff
@@ -2158,15 +2158,19 @@ const Gpu = class SystemMonitor_Gpu extends ElementBase {
         this.menu_items[4].text = unit;
     }
     _apply() {
+        this.tip_unit_labels[1].text = "/ " + this.total + " " + this.menu_items[4].text;
         if (this.total === 0) {
-            this.vals = [0];
-            this.tip_vals = [0];
+            this.vals = [0, 0];
+            this.tip_vals = [0, 0];
         } else {
-            this.vals = [this.percentage];
-            this.tip_vals = [Math.round(this.vals[0])];
+            // we subtract percentage from memory because we do not want memory to be 
+            // "accumulated" in the chart with utilization; these two measures should be 
+            // independent
+            this.vals = [this.percentage, this.mem / this.total * 100 - this.percentage];
+            this.tip_vals = [Math.round(this.vals[0]), this.mem];
         }
-        this.text_items[0].text = this.tip_vals.toString();
-        this.menu_items[0].text = this.tip_vals.toLocaleString();
+        this.text_items[0].text = this.tip_vals[0].toString();
+        this.menu_items[0].text = this.tip_vals[0].toLocaleString();
```

Comment:
The toLocaleString() method needs the locale as argument now:
this.menu_items[0].text = this.tip_vals[0].toLocaleString(Locale);

Reply:
resolved while merging with master